### PR TITLE
XUPVVH: Network Feature: Add Aurora backend as alternative to Ethernet

### PR DESCRIPTION
**Pull request by mhrtmnn**

### CODE REVIEW

File: toolflow/vivado/common/common_2019.2.tcl [hunk 1]
```diff
@@ -26,5 +26,6 @@ dict set stdcomps   mig_core         vlnv   "xilinx.com:ip:mig_7series:4.2"
 dict set stdcomps   ultra_ps         vlnv   "xilinx.com:ip:zynq_ultra_ps_e:3.3"
 dict set stdcomps   xxv_ethernet     vlnv   "xilinx.com:ip:xxv_ethernet:3.1"
 dict set stdcomps   100g_ethernet    vlnv   "xilinx.com:ip:cmac_usplus:3.0"
+dict set stdcomps   aurora           vlnv   "xilinx.com:ip:aurora_64b66b:12.0"
```
Comment (wirthjohannes) [inline toolflow/vivado/common/common_2019.2.tcl:29]: You always need to add this for all relevant Vivado versions. So in this case at least 2019.1 and 2019.2 for XUPVVH

**Review by wirthjohannes — CHANGES_REQUESTED** [review]
Mostly good, just this single point.
Also please merge the latest develop-version so the CI works

File: toolflow/vivado/common/common_2019.2.tcl (hunk 1 above)
Comment (mhrtmnn) [reply toolflow/vivado/common/common_2019.2.tcl:29]: Thanks, fixed

**Review by wirthjohannes — APPROVED** [review]
Looks good

**Pull request merged**